recipes/site_pr_builder.rb

```diff
@@ -0,0 +1,49 @@
+#
+# Cookbook:: osl-jenkins
+# Recipe:: github_comment
+#
+# Copyright:: 2017, Oregon State University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+include_recipe 'osl-jenkins::master'
+include_recipe 'osl-jenkins::github_comment'
+
+site_pr_builder = node['osl-jenkins']['site_pr_builder']
+
+site_pr_builder['sites_to_build'].each do |site, site_url|
+  job_name = site + '_pr_builder'
+  rsync_target = "/var/www/staging.osuosl.org/#{site}-$ghprbPullId"
+  pelican_site_url = "http://#{site}-$ghprbPullId.staging.osuosl.org"
```

Comment:
I don't believe sphinx needs to be given a site_url, it's a pelican thing only (so it can link to its own theme assets). We can call it site_url instead, I suppose, just in case we have a use for it in other static sites later.